src/vue-components/panel.vue

```diff
@@ -83,8 +83,10 @@
               </alph-tooltip>
             </span>
         </div>
-
+        <div class="alpheios-panel__header" style="grid-area: b;"><reskin-font-color></reskin-font-color></div>
```

Comment:
It's better not use inline styles (this and the next line). If we need to modify behavior, we should use class modifiers such as `alpheios-panel__header--mod-description`. Modifier classes are much easier to maintain.

Reply:
Yes, I agree - I used it for testing purpose and forgot to move to styles.